src/main/webapp/constants/constant.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    MAX_SIZE: '512000'
```

Comment:
I think we can make it a bit specific to the file as well, what do you think about `FILE_MAX_SIZE`?